### ISSUE TITLE
Method Error in Notebook  example

### ISSUE THREAD

**Issue by jmatsushita** [AlgebraicJulia/AlgebraicRelations.jl#34]
Hi there,

I'm trying to run [this example notebook](https://github.com/AlgebraicJulia/AlgebraicRelations.jl/blob/master/examples/ml_workflow_demo/ml_demo.ipynb) in VS Code. I'm using the Julia 1.8.3 kernel and I've just did `add AlgebraicRelations` in the package manager. 

When I run the second cell (the `using` first cell runs) I get the following error:
```
MethodError: no method matching present_to_schema(::Presentation{Catlab.Theories.ThSymmetricMonoidalCategory, Symbol})
Closest candidates are:
  present_to_schema(!Matched::Symbol, !Matched::Symbol) at ~/.julia/packages/AlgebraicRelations/VrfVu/src/Presentations.jl:48

Stacktrace:
 [1] top-level scope
   @ ~/dev/workshop/julia/AlgebraicRelations.jl/examples/ml_workflow_demo/ml_demo.ipynb:18
```

I'm new to both Julia and the Catlab ecosystem, so apologies if I've missed something obvious!

Cheers,

Jun

**Comment by jmatsushita** [comment]
I think I found the problem. 
 * Looking at [this test](https://github.com/AlgebraicJulia/AlgebraicRelations.jl/blob/master/test/Presentations.jl#L21) it seems that the way to call `present_to_schema` is to use the `macro` 
 * If I replace `TrainDB = present_to_schema(present);` with `@present_to_schema TrainDB(present);` the cell runs correctly and draws the schema.
 
I'll submit a PR to fix the notebook.

**Comment by epatters** [comment]
Hi, thanks for tracking down this problem and suggesting a fix. This package has been without a maintainer for a little while, but @TheCedarPrince is going to start working on it, beginning with the big refactor in #31. It may take awhile to triage these issues. Thanks for your interest and your patience!

**Comment by TheCedarPrince** [comment]
Yup! I am ramping up on this and will come back to this error @jmatsushita -- sorry you ran into problems! Like @epatters , I'll be picking up with a major refactor and then can revisit not only this issue but others that are outstanding. Thanks for coming through! If you have any more questions, feel free to also follow-up with us in [Zulip](https://julialang.zulipchat.com/#narrow/stream/230248-catlab.2Ejl)
